src/jit/compiler.h

```diff
@@ -8107,15 +8107,11 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     {
 #ifdef FEATURE_SIMD
         unsigned vectorRegSize = getSIMDVectorRegisterByteLength();
-        if (vectorRegSize > TARGET_POINTER_SIZE)
-        {
-            return vectorRegSize;
-        }
-        else
-#endif // FEATURE_SIMD
-        {
-            return TARGET_POINTER_SIZE;
-        }
+        assert(vectorRegSize > TARGET_POINTER_SIZE);
```

Comment:
It is probably reasonable to assume that a vector register will always be at least as large as the target registers, but perhaps not that it will always be larger. The result is the same if you change this to `>=` and seems the better alternative.

Reply:
Done.